src/server.mjs

```diff
@@ -113,6 +114,36 @@ var limits = {
 };
 var upload = multer({ storage, limits });
 
+// runs every hour
+// deletes any files in the download directory past the set fileExpirationTime set in config
+cron.schedule('0 * * * *', () => {
+  console.log("cron scheduled check");
+  const currentTime = new Date().getTime();
+  // loop through these specified directories
+  for (const dir of [DOWNLOAD_DATA_PATH, UPLOAD_DATA_PATH]) {
+    fs.readdir(dir, (err, files) => {
```

Comment:
This only handles one flat directory, right? Don't we download files to different subdirectories?

We have an `mkdirSync()` here:
https://github.com/vgteam/sequenceTubeMap/blob/0280401b5ab6da396231a06a2c9b3f6d6596418b/src/server.mjs#L1285-L1288

But this cleanup code has nothing to loop through the subdirectories recursively, and no `rmdir` to remove the emptied directories.